### DISCUSSION TITLE
DONE: Hide Overlay if focus window name does not match "WOB #" or related

### DISCUSSION THREAD

**Discussion by I3igAl** [WowOpenBox/WowOpenBox#98] [Ideas]
The new feature for Ver 3.9 does not work very good IMO because if you have a full screen window like Discord or even a Youtube Video, if there is a WOB overlay area in that part of the screen it still catches the mouse. if I alt tab to Discord, or click on Chrome, the focus changes. testing window focus would be more reliable for hiding overlay, and probably easier than watching mouse position.

**Comment by mooreatv** [comment]
done in 4.2!
